### ISSUE TITLE
Build Setting for Configuration

### ISSUE THREAD

**Issue by mrgrauel** [0xced/xcproj#10]
Is it possible to read a build setting for a specified configuration?

``` bash
xcproj read-build-setting PRODUCT_BUNDLE_IDENTIFIER -c=Release
```

**Comment by landfound** [comment]
same issue

**Comment by 0xced** [comment]
I haven’t tagged a new version yet but it’s available on develop since 22e509d7c0929d7858bfecbcda46790086b45e74.

**Comment by 0xced** [comment]
The `--configuration` option is now available in version 0.2.
